heat/core/statistics.py

```diff
@@ -1126,14 +1125,14 @@ def reduce_vars_elementwise(output_shape_i):
             return dndarray.DNDarray(ret, tuple(ret.shape), types.canonical_heat_type(ret.dtype), None, x.device, x.comm)
 
         else:  # case for full matrix calculation (axis is None)
-            if x.lshape[x.split] != 0:
-                mu_in = operations.__local_op(torch.mean, x, out=None)
-                var_in = operations.__local_op(torch.var, x, out=None, unbiased=bessel)
-                if torch.isnan(var_in._DNDarray__array):
-                    var_in = 0.0
-            else:
-                mu_in = 0
-                var_in = 0
+            mu_in = operations.__local_op(torch.mean, x, out=None)
```

Comment:
should also probably not be a local op

Reply:
Would you like to have this cases of "wrong" usage of the local op's removed?